### PR TITLE
Cleanup for edging to most updated within hapi 16

### PR DESCRIPTION
**Pull request by zemccartney**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
     "topo": "2.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=10 <17"
+    "hapi": "16.x.x"
```
Comment (zemccartney) [inline package.json]: Wasn't sure about this, guessed based on Eran's note about supporting v16 in the v17 release notes.

Comment (devinivy) [inline package.json]: This was correct as-is– we support hapi v10 thru 16.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hodgepodge",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (devinivy) [inline package.json]: No need to bump the version– I'll do that :)

File: package.json (hunk 1 above)
Comment (zemccartney) [reply package.json]: werd! will update. thanks, dude!

File: package.json (hunk 2 above)
Comment (zemccartney) [reply package.json]: 👍  🙏

File: package.json (hunk 1 above)
Comment (devinivy) [reply package.json]: We should move it back to how it was, but no problem!  My rationale is this: dropping support for hapi 10-15 would be a breaking change, and there's no real advantage to make that change.  In hodgepodge v3 we'll drop support for 10-16.

**Pull request merged**